homeassistant/components/isy994/entity.py

```diff
@@ -53,6 +57,53 @@ def on_control(self, event: NodeProperty) -> None:
 
         self.hass.bus.fire("isy994_control", event_data)
 
+    @property
+    def device_info(self):
+        """Return the device_info of the device."""
+        if hasattr(self._node, "protocol") and self._node.protocol == PROTO_GROUP:
+            # not a device
+            return None
+        uuid = self._node.isy.configuration["uuid"]
+        node = self._node
+        basename = self.name
+
+        if hasattr(self._node, "parent_node") and self._node.parent_node is not None:
+            # This is not the parent node, get the parent node.
+            node = self._node.parent_node
+            basename = node.name
+
+        device_info = {
+            "name": basename,
+            "identifiers": {},
+            "model": "Unknown",
+            "manufacturer": "Unknown",
+            "via_device": (DOMAIN, uuid),
+        }
+
+        if hasattr(node, "address"):
+            device_info["name"] += f" ({node.address})"
+        if hasattr(node, "primary_node"):
+            device_info["identifiers"] = {(DOMAIN, f"{uuid}_{node.address}")}
+        # ISYv5 Device Types
+        if hasattr(node, "node_def_id") and node.node_def_id is not None:
+            device_info["model"] = node.node_def_id
+            # Numerical Device Type
+            if hasattr(node, "type") and node.type is not None:
+                device_info["model"] += f" {node.type}"
+        if hasattr(node, "protocol"):
+            device_info["manufacturer"] = node.protocol
+            if node.protocol == PROTO_ZWAVE:
+                # Get extra information for Z-Wave Devices
+                device_info["manufacturer"] += f" mfr_id:{node.zwave_props.mfr_id}"
+                device_info["model"] += (
+                    f" Type:{node.zwave_props.mfr_id} "
+                    f"ProductTypeID:{node.zwave_props.prod_type_id} "
```

Comment:
```suggestion
                    f" ProductTypeID:{node.zwave_props.prod_type_id} "
```

Looks like we already have this in `manufacturer`?

Reply:
Changed to the General Device Type string. Think it was a copy paste error way back when.

Reply:
Missed it the first 2 reviews of this but small chunks = 👌